src/MICore/ExampleLaunchOptions.xml

```diff
@@ -66,8 +66,4 @@
     TargetArchitecture="x86"
     AdditionalSOLibSearchPath=""/>
 
-  <SSHLaunchOptions xmlns="http://schemas.microsoft.com/vstudio/MDDDebuggerOptions/2014"
```

Comment:
>SSHLaunchOptions [](start = 3, length = 16)

nit: this was the only example of SSHLaunchOptions, so might want to instead make it a GDB one...